Makefile

```diff
@@ -2,6 +2,9 @@ BIN_OUTPUT_PATH = bin/$(shell uname -s)-$(shell uname -m)
 
 TOOL_BIN = bin/gotools/$(shell uname -s)-$(shell uname -m)
 
+NDK_ROOT = ~/android-ndk-r25c
+SERVER_RELEASE = dev
```

Comment:
For all the other binaries, we use "local" instead of "dev" as the keyword for the "BUILD_CHANNEL" variable. (The other words there are "latest" or "stable") I think this should use the same variable/words. See packaging.make and maybe consider following the paradigm there.

Reply:
changed to BUILD_CHANNEL and local in 646adbac